client/blocks/announcement-modal/style.scss

```diff
@@ -0,0 +1,204 @@
+
+// Core modal style overrides
+.announcement-modal {
+	&.components-modal__frame {
+		overflow: visible;
+		height: 65vh;
+		top: calc( 17.5vh - #{$modal-footer-height * 0.5} );
+
+		@media ( max-width: $modal-breakpoint ) {
+			width: 90vw;
+			min-width: 90vw;
+			left: 5vw;
+			right: 5vw;
+		}
+
+		@media ( min-width: $modal-breakpoint ) {
+			width: 740px;
+			height: $modal-content-min-height;
+			top: calc( 50% - #{$modal-footer-height * 0.5} );
+		}
+
+		@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+			height: $modal-content-min-height-landscape;
+			top: 50%;
+		}
+	}
+
+	* {
+		box-sizing: border-box;
+	}
+
+	.components-modal__header {
+		display: block;
+		position: absolute;
+		text-align: right;
+
+		svg {
+			fill: white;
+			filter: drop-shadow( 0px 0px 1px black );
+		}
+
+		@media ( min-width: $modal-breakpoint ) {
+			position: fixed;
+
+			svg {
+				fill: currentColor;
+				filter: none;
+			}
+		}
+	}
+
+	.components-guide__footer {
+		display: none;
+	}
+	.components-guide__container {
+		margin-top: 0;
+	}
+
+	.components-guide__page {
+		@media ( min-width: $modal-breakpoint ) {
+			height: $modal-content-min-height;
+		}
+
+		@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+			height: $modal-content-min-height-landscape;
+		}
+	}
+}
+
+.announcement-modal__page {
+	display: flex;
+	flex-direction: column-reverse;
+	justify-content: flex-end;
+	background: white;
+	width: 100%;
+	overflow: hidden;
+
+	@media ( min-width: $modal-breakpoint ) {
+		flex-direction: row;
+		justify-content: flex-start;
+		position: absolute;
+		min-height: $modal-content-min-height;
+		height: 100%;
+		bottom: 0;
+	}
+
+	@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+		min-height: $modal-content-min-height-landscape;
+	}
+}
+
+.announcement-modal__headline {
+	font-size: 10px;
+	font-weight: 700;
+	color: var( --studio-blue-40 );
+}
+
+.announcement-modal__text,
+.announcement-modal__visual {
+	@media ( min-width: $modal-breakpoint ) {
+		flex: 1 auto;
+		min-width: 355px;
+	}
+}
+
+.announcement-modal__text {
+	padding: 25px;
+	height: 60%;
+
+	@media ( min-width: $modal-breakpoint ) {
+		height: auto;
+		padding: $modal-padding-v $modal-padding-h;
+	}
+
+	@media ( min-width: $modal-breakpoint ) and ( max-height: $modal-breakpoint-phone-height ) {
+		padding: 25px;
+	}
+}
+.announcement-modal__visual {
+	height: 95%;
+	align-items: center;
+	justify-content: center;
+	background: var( --studio-blue-5 );
+	display: flex;
+	text-align: center;
+	overflow: hidden;
+	position: relative;
+
+	@media ( min-width: $modal-breakpoint ) {
+		height: auto;
+	}
+}
+
+.announcement-modal__picture {
+	max-height: 100%;
+
+	@media ( min-width: $modal-breakpoint ) {
+		margin-top: 0;
+		width: 90%;
+	}
+}
+
+.announcement-modal__heading {
+	@include onboarding-font-recoleta;
+	/* Gray / Gray 90 */
+	color: #1d2327;
```

Comment:
Nitpicking: Any reason for not using the ` --studio-gray-90` variable?

Reply:
46bf8a3